Library/Homebrew/extend/os/linux/extend/ENV/super.rb

```diff
@@ -31,6 +31,8 @@ def setup_build_environment(formula: nil, cc: nil, build_bottle: false, bottle_a
     self["HOMEBREW_RPATH_PATHS"] = determine_rpath_paths(@formula)
     m4_path_deps = ["libtool", "bison"]
     self["M4"] = "#{HOMEBREW_PREFIX}/opt/m4/bin/m4" if deps.any? { m4_path_deps.include?(_1.name) }
+
+    append_to_cccfg "b" if Hardware::CPU.arch == :arm64 && DevelopmentTools.gcc_version("gcc") >= 9
```

Comment:
```suggestion
    append_to_cccfg "b" if Hardware::CPU.arm? && DevelopmentTools.gcc_version("gcc") >= 9
```
Maybe? Though I guess this might be true for aarch32.

Reply:
yeah `arm?` checks the type and that would be incorrect for aarch32

Reply:
```suggestion
    # some comment explaining the motivation for this
    append_to_cccfg "b" if Hardware::CPU.arch == :arm64 && DevelopmentTools.gcc_version("gcc") >= 9
```